python/servo/package_commands.py

```diff
@@ -674,6 +677,22 @@ def upload_to_s3(platform, package, timestamp):
                 }
             )
 
+            if github_release_id:
+                from github import Github
```

Comment:
I would move this to the top of the Python script. Typically, these inner use statements are only used when the package isn't always available.

Reply:
Done in 61fb795